src/pages/Lending/hooks/useLendingSupportedAssets/index.ts

```diff
@@ -22,7 +22,13 @@ import { store, useAppSelector } from 'state/store'
 
 const queryKey = ['lendingSupportedAssets']
 
-export const useLendingSupportedAssets = ({ type }: { type: 'collateral' | 'borrow' }) => {
+export const useLendingSupportedAssets = ({
+  type,
+  statusFilter = ['Available'],
+}: {
+  type: 'collateral' | 'borrow'
+  statusFilter?: string[]
```

Comment:
suggestion: if we're only ever filtering by one criteria, this could be a string instead

question: do we see a case where we will ever have to filter by a more advanced criteria than a single status? If so, it may make sense to pass an optional filter callback instead

Reply:
Alright [8adb4bc](https://github.com/shapeshift/web/pull/7191/commits/8adb4bc5d03daab4737af445ebe41feedd9d88de)